index.js

```diff
@@ -25,10 +25,10 @@ BrowserSyncPlugin.prototype.apply = function (compiler) {
 
   compiler.plugin('done', function (stats) {
     if (stats.hasErrors() || stats.hasWarnings()) {
+      var error = stats.toString('minimal')
       return self.browserSync.sockets.emit('fullscreen:message', {
         title: 'Webpack Error:',
-        body: stripAnsi(stats.toString()),
-        timeout: 100000
+        body: stripAnsi(error),
       })
     }
     self.browserSync.reload()
```

Comment:
I agree with you for timeout part, IMO, it can be hard coded to infinite timeout.
Personally, if you want to remove the error message, I fix my error :smile: